cmd/zc_traverser_file.go

```diff
@@ -34,6 +34,8 @@ import (
 	"github.com/Azure/azure-storage-azcopy/v10/common"
 )
 
+const trailingDotErrMsg = "File share contains file/directory %s with a trailing dot but the trailing dot flag was not set to true. AzCopy will automatically filter this file/directory out to prevent data corruption."
```

Comment:
Given we're now raw yes/no and not strip/include/filter, is this message accurate or necessary?

Reply:
You're right, let me fix the verbiage.